l3kernel/l3int.dtx

```diff
@@ -1386,7 +1387,7 @@
       }
     #2 = \@@_eval:w #1 \@@_eval_end:
   }
-\cs_generate_variant:Nn \int_const:Nn { c }
```

Comment:
I think removing this is wrong here.

Reply:
I just moved where it is a bit: now on line 1369.